cloudsmith_cli/cli/commands/policy/vulnerability.py

```diff
@@ -110,6 +112,17 @@ def ls(ctx, opts, owner, page, page_size):
 
     print_vulnerability_policies(policies)
 
+    click.echo()
```

Comment:
does the newline break JSON parsing if you use JSON output?

Reply:
Unresolved this one.

Reply:
☝️ Still need an answer here